src/sentry/static/sentry/app/views/issueList/actions.jsx

```diff
@@ -562,12 +563,30 @@ const IssueListActions = createReactClass({
               </GraphToggle>
             </GraphHeader>
           </GraphHeaderWrapper>
-          <EventsOrUsersLabel className="align-right">
-            <ToolbarHeader>{t('Events')}</ToolbarHeader>
-          </EventsOrUsersLabel>
-          <EventsOrUsersLabel className="align-right">
-            <ToolbarHeader>{t('Users')}</ToolbarHeader>
-          </EventsOrUsersLabel>
+          <ToolbarHeader>
+            <EventsOrUsersLabel className="align-right">
+              <React.Fragment>
+                {t('Events')}
+                <QuestionTooltip
```

Comment:
Is `QuestionTooltip` a styled component? if so we can use it as a selector and use our breakpoints below to hide this so we can get away from bootstrap's grid.

Reply:
It was not a styled component, but i did switch it to a media selector instead of the bootstrap grid